village/src/App.js

```diff
@@ -11,14 +13,48 @@ class App extends Component {
       smurfs: [],
     };
   }
+
+  componentDidMount() {
+   this.getSmurfs()
+  }
+
+  getSmurfs() {
+    axios
+    .get(`${baseUrl}/smurfs`)
+    .then(res => 
+      this.setState({
+        smurfs: res.data
+      }))
+
+    .catch(err => console.log(err))
+
+  }
+
+  deleteSmurf = (ev, smurfId) => {
+    ev.preventDefault()
+    axios.delete(`${baseUrl}/smurfs/${smurfId}`)
+      .then(res => this.setState({
+      smurfs: res.data
+      }))
+    .catch(err => console.log(err))
+  }
+
   // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
   // Notice what your map function is looping over and returning inside of Smurfs.
   // You'll need to make sure you have the right properties on state and pass them down to props.
   render() {
     return (
       <div className="App">
-        <SmurfForm />
-        <Smurfs smurfs={this.state.smurfs} />
+        <nav>
+          <h1 className="header">Smurfs Village!!</h1>
+        <div className="nav-links"> 
+            <NavLink onClick={() => this.getSmurfs()} to="/">Smurfs List!</NavLink>
```

Comment:
Solution works but is a bit hacky. Try to minimize the number of `axios.get`s to optimize performance.